allensdk/brain_observatory/chisquare_categorical.py

```diff
@@ -84,12 +84,12 @@ def stim_table_to_categories(stim_table,
 
     category = 0
     sweep_categories = -1*np.ones((num_sweeps,))
-    curr_combination = np.zeros((num_params,),dtype=np.int)
-    options_per_column = np.array(options_per_column).astype(np.int)
+    curr_combination = np.zeros((num_params,),dtype='int')
+    options_per_column = np.array(options_per_column).astype('int')
     all_tried = False
     while not all_tried:
         
-        matches_combination = np.ones((num_sweeps,),dtype=np.bool)
+        matches_combination = np.ones((num_sweeps,),dtype=bool)
```

Comment:
You seem to input types besides `bool` as a string to `dtype`. Is there are a reason for this?

Reply:
float/int sizes (eg, float32 vs float64) can be automatically managed by defining it as a string. Bool does not have any variations.